MapboxCoreNavigation/RouteController.swift

```diff
@@ -32,19 +32,23 @@ open class RouteController: NSObject {
     */
     public var snapsUserLocationAnnotationToRoute = false
     
+    var simulate: Bool = false
     
     /*
      Intializes a new `RouteController`.
      
      - parameter Route: A `Route` object representing the users route it will follow.
      */
-    public init(route: Route) {
+    public init(route: Route, simulate: Bool = false) {
```

Comment:
Argument labels shouldn’t be verbs either. How about `simulatingLocationUpdates` as the external label and `simulatesLocationUpdates` as the internal variable name?